locustfiles/prime.py

```diff
@@ -27,7 +27,6 @@ class PrimeUser(MilMoveHostMixin, HttpUser):
     # These are locust HttpUser attributes that help define and shape the load test:
     wait_time = between(0.25, 9)  # the time period to wait in between tasks (in seconds, accepts decimals and 0)
     tasks = {PrimeTasks: 1}  # the set of tasks to be executed and their relative weight
-    weight = 5
```

Comment:
Why are the weights no longer needed?

Reply:
The weights were throwing off the balance of PrimeUsers vs. SupportUsers. When we still had them, Locust would spawn 5 PrimeUsers to every 1 SupportUser. This meant I'd need to spawn 6 users if I wanted the support tasks to run at all. At first I thought it made sense because we wanted to test the Prime API more than the Support API, but it just gets in the way of development now. I want to be able to run a load test with 2 users and make sure one is Prime and one is Support